utils/_context/_scenarios/__init__.py

```diff
@@ -470,10 +470,18 @@ def all_endtoend_scenarios(test_object):
         doc="Scenario with custom headers for DD_TRACE_HEADER_TAGS that libraries should reject",
     )
 
+    tracing_config_empty =EndToEndScenario(
+        "TRACING_CONFIG_EMPTY", weblog_env={}, doc="",
+    )
+
     tracing_config_nondefault = EndToEndScenario(
         "TRACING_CONFIG_NONDEFAULT", weblog_env={"DD_TRACE_HTTP_SERVER_ERROR_STATUSES": "200-201,202"}, doc="",
     )
 
+    tracing_config_nondefault_client_tag_query = EndToEndScenario(
+        "TRACING_CONFIG_NONDEFAULT_CLIENT_TAG_QUERY", weblog_env={"DD_HTTP_CLIENT_TAG_QUERY_STRING": "false"}, doc="",
+    )
```

Comment:
@cbeauchesne , we're running into an issue where DD_HTTP_CLIENT_TAG_QUERY_STRING and DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP interact when configured so we need to test them separately, meaning we need yet another, separate scenario. I know that's typically a problem. Do you have any suggestions?

Reply:
Is it supposed to interact that way ? 

If yes, let's continue with `tracing_config_nondefault_3` 😓 (see #3007)

Reply:
yes, sorry, I wasn't clear enough.